lib/src/tag_detection.dart

```diff
@@ -701,4 +762,15 @@ class Tagger {
     }
     return result;
   }
+
+  List<String> nullSafetyTags() {
+    final nullSafety = NullSafety(_packageGraph, _pubspecCache, _session);
+    final nullSafetyResult = nullSafety.findNullSafetyViolation(packageDir);
+    if (nullSafetyResult.hasPath) {
+      log.info(
+          '$packageDir is not null-safety compliant because of ${nullSafetyResult.path}');
+      return [];
+    }
+    return ['is:null-safety'];
```

Comment:
How about `is:null-safe`, we are going to surface it differnetly in the UI, anyways.

We could also do `is:null-safety-compliant`, but that just seems very long.

Reply:
Sure - I intuitively prefer that. Only I'm learning from @mit-mit that that is the wrong name for the feature.

Reply:
I think I'm ok with `is:null-safe` here; the alternatives seem worse